src/nvim/api/win_config.c

```diff
@@ -295,6 +295,10 @@ Window nvim_open_win(Buffer buffer, Boolean enter, Dict(win_config) *config, Err
     goto cleanup;
   }
 
+  if (fconfig._cmdline_offset < INT_MAX) {
```

Comment:
In the `nvim_open_win` we should mention the flag and mention "EXPERIMENTAL". We don't have a better way of documenting experiemental stuff currently. But not documenting it at all means it could be forgotten and/or lead to confusion in the future.